### PR TITLE
[ADD] Accounting: Bank Statements and remove old documentation

### PR DESCRIPTION
**Pull request by jcs-odoo**
There were documentation pages in the accounting chapter that were explaining how to import
bank files or how to record them manually. This new documentation replaces them all.

### CODE REVIEW

File: accounting/bank/feeds/bank_statements.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+============================
+Odoo supports multiple file formats to import bank statements:
+
+- SEPA recommended Cash Management format (CAMT.053)
+- Comma-separated values (.CSV)
+- Open Financial Exchange (.OFX)
+- Quicken Interchange Format (.QIF)
+- Belgium Coded Statement of Account (.CODA)
+
+to import them, go to :menuselection:`Accounting --> Overview --> Bank`, click on *Import
```
Comment (Lau-Odoo) [inline accounting/bank/feeds/bank_statements.rst]: Uppercase at the beginning of a sentence :D

File: accounting/bank/feeds/bank_synchronization.rst [hunk 2]
```diff
@@ -125,4 +125,7 @@ You can update your credentials in the *Developer Mode*.
 
 To do so, go to :menuselection:`General Settings`, click on *Activate developer mode*, then go to
 :menuselection:`Accounting --> Configuration --> Online Synchronization`, then open the Institution
```
Comment (Lau-Odoo) [inline accounting/bank/feeds/bank_synchronization.rst]: Two "then" in the same sentence :)

File: accounting/bank/feeds/bank_statements.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Bank Statements
+===============
+Importing your bank statements in Odoo Accounting allows you to keep track of the financial
+movements that occur on your bank accounts and reconcile them with the transactions recorded in your
+accounting.
+
+The easiest way to do so is by synchronizing. To do so, please read the related documentation:
+:doc:`bank_synchronization`.
+
+However, if your bank account is not synchronized with Odoo, you still have two options:
```
Comment (cdu-odoo) [inline accounting/bank/feeds/bank_statements.rst]: Forgot something

Comment (jcs-odoo) [reply accounting/bank/feeds/bank_statements.rst]: I don't think so :)
#test review

**Comment by jubodoo** [comment]
@jcs-odoo Ta un conflict faut que tu le resolve.

**Pull request merged**